nservicebus/messaging/index.md

```diff
@@ -1,4 +1,48 @@
 ---
 title: Messaging
 summary: Outline the various approach to sending-receiving, defining messages and common messaging patterns.
+reviewed: 2016-11-29
 ---
+
+Messages are discrete units of data communicated to one or more recipients using a predefined contract. Messages are a common way for systems or applications to interact, indeed the most common communications protocols on the Internet use messaging at their heart. HTTP protocol describes both a request message and the response message. Even SMTP is a messaging based system.
+
+Communication via messaging occurs either directly, as in the case of HTTP, or indirectly in the case of SMTP. The difference being whether the producer and/or consumer of the message is blocked during transfer and/or consumption of the message.
+
+NServiceBus provides a means and standards for implementing indirect communication of messages between processes in .NET.
+
+## Physical, Logical, and Native Messages
+
+Messsaging protocols often make a distinction between a physical message and the logical message.
+
+The native message is the protocol/transport specific message format required by the underlying messaging technology.
+
+The physical message contains the logical message as well message metadata.
```

Comment:
This is the Transport Message in NServiceBus core, but is "physical" message the right descriptor? Or is it a synonym for the Native message (e.g. BrokeredMessage in ASB)?